src/main/scala/viper/gobra/translator/implementations/translator/BuiltInMembersImpl.scala

```diff
@@ -385,6 +423,207 @@ class BuiltInMembersImpl extends BuiltInMembers {
           in.Access(closedInst, in.FullPerm(src))(src)
         )
         in.Function(x.name, args, Vector(), pres, posts, None)(src)
+
+      /* João, 18/08/2021:
+       *  The spec for `append` currently does not allow the first and second non-ghost arguments to be the same. The go
+       *  spec however allows that to happen and the behavior is well-defined for those cases (in particular, the result
+       *  of appending two slices is independent of whether they overlap). I did not change the spec to reflect this change
+       *  at the moment to avoid surprises in the performance of Gobra when checking the VerifiedSCION codebase. I do expect
+       *  to change that at a later point.
```

Comment:
One idea to hopefully better control performance could be an `overlap` ghost flag that turns "enables" / "disables" one set of spec or the other

Reply:
I may try that in a future PR. I think we should close this one now, given that it has been open for quite a while.